src/arqa.py

```diff
@@ -117,6 +118,112 @@ def arc_fit_qa(slf, fit, outfil=None, ids_only=False, title=None):
     return
 
 
+def flexure(slf, det, flex_dict):
+    """ QA on flexure measurement
+
+    Parameters
+    ----------
+    slf
+    det
+    flex_dict
+
+    Returns
+    -------
+
+    """
+    # Setup
+    nobj = len(slf._specobjs[det-1])
+    if nobj == 0:
+        return
+    ncol = min(3,nobj)
+    nrow = nobj // ncol + ((nobj%ncol) > 0)
+
+
+    plt.figure(figsize=(8, 5.0))
+    plt.clf()
+    gs = gridspec.GridSpec(nrow, ncol)
+
+    # Correlation QA
+    for o,specobj in enumerate(slf._specobjs[det-1]):
+        ax = plt.subplot(gs[o//ncol, o%ncol])
+        # Fit
+        fit = flex_dict['polyfit'][o]
+        xval = np.linspace(-10., 10, 100) + flex_dict['corr_cen'][o] + flex_dict['shift'][o]
+        model = (fit[2]*(xval**2.))+(fit[1]*xval)+fit[0]
+        mxmod = np.max(model)
+        ylim = [np.min(model/mxmod), 1.5]
+        ax.plot(xval-flex_dict['corr_cen'][o], model/mxmod, 'k-')
+        # Measurements
+        ax.scatter(flex_dict['subpix'][o]-flex_dict['corr_cen'][o],
+                   flex_dict['corr'][o]/mxmod, marker='o')
+        # Final shift
+        ax.plot([flex_dict['shift'][o]]*2, ylim, 'g:')
+        # Label
+        ax.text(0.04, 0.25, '{:s}'.format(specobj.idx), transform=ax.transAxes, size='large', ha='left')
+        ax.text(0.04, 0.15, 'flex_shift = {:g}'.format(flex_dict['shift'][o]),
+                transform=ax.transAxes, size='large', ha='left')#, bbox={'facecolor':'white'})
+        # Axes
+        ax.set_ylim(ylim)
+        ax.set_xlabel('Lag')
+
+    # Finish
+    plt.tight_layout(pad=0.2, h_pad=0.0, w_pad=0.0)
+    slf._qa.savefig(bbox_inches='tight')
+    plt.close()
+
+    # Sky line QA (just one object)
+    o=0
+    specobj = slf._specobjs[det-1][o]
+    sky_spec = flex_dict['sky_spec'][o]
+    arx_spec = flex_dict['arx_spec'][o]
+
+    # Sky lines
+    sky_lines = np.array([3370.0, 3914.0, 4046.56, 4358.34, 5577.338, 6300.304,
+              7340.885, 7993.332, 8430.174, 8919.610, 9439.660,
+              10013.99, 10372.88])*u.AA
+    dwv = 20.*u.AA
+    gdsky = np.where((sky_lines>sky_spec.wvmin) & (sky_lines < sky_spec.wvmax))[0]
+    if len(gdsky) == 0:
+        msgs.warn("No sky lines for Flexure QA")
+        return
+    if len(gdsky) > 6:
+        idx = np.array([0,1,len(gdsky)/2,len(gdsky)/2+1,-2,-1])
+        gdsky = gdsky[idx]
+
+    # Figure
+    plt.figure(figsize=(8, 5.0))
+    plt.clf()
+    nrow, ncol = 2, 3
+    gs = gridspec.GridSpec(nrow, ncol)
+    plt.suptitle('Sky Comparison for {:s}'.format(specobj.idx))
```

Comment:
Perhaps add the keyword argument `y=1.05` (you may need to tweak this value a little), so that the title doesn't overlap the middle panel?